src/App.css

```diff
@@ -177,3 +177,20 @@ body {
     color: grey !important;
   }
 }
+
+.footer {
+  background: white !important;
+  padding: 9px 0px !important;
+  position: fixed !important;
+  left: 0;
+  bottom: 0;
+  width: 100%;
```

Comment:
for safety might want to throw a `z-index` on this with a pretty high value, this way we guarantee it stays on the top layer